diff -up gnucash-2.4.5/configure.ac.foo gnucash-2.4.5/configure.ac
--- gnucash-2.4.5/configure.ac.foo	2011-05-13 14:32:12.403684951 -0400
+++ gnucash-2.4.5/configure.ac	2011-05-13 14:33:28.397684863 -0400
@@ -410,7 +410,7 @@ if test "x$saved_GUILE_INCS" != x; then
   GUILE_INCS="$saved_GUILE_INCS"
 fi
 
-AS_SCRUB_INCLUDE(GUILE_INCS)
+AC_SUBST(GUILE_INCS)
 AC_SUBST(GUILE_LIBS)
 AC_SUBST(GUILE)
 
diff -up gnucash-2.4.5/src/import-export/aqbanking/Makefile.in.foo gnucash-2.4.5/src/import-export/aqbanking/Makefile.in
--- gnucash-2.4.5/src/import-export/aqbanking/Makefile.in.foo	2011-05-13 14:51:07.793685262 -0400
+++ gnucash-2.4.5/src/import-export/aqbanking/Makefile.in	2011-05-13 14:51:57.459685276 -0400
@@ -480,6 +480,7 @@ AM_CFLAGS = \
   ${GNOME_CFLAGS} \
   ${GLADE_CFLAGS} \
   ${GLIB_CFLAGS} \
+  ${GUILE_CFLAGS} \
   ${AQBANKING_CFLAGS}
 
 uidir = $(GNC_UI_DIR)
diff -up gnucash-2.4.5/src/plugins/bi_import/Makefile.in.foo gnucash-2.4.5/src/plugins/bi_import/Makefile.in
--- gnucash-2.4.5/src/plugins/bi_import/Makefile.in.foo	2011-05-13 14:53:18.470685304 -0400
+++ gnucash-2.4.5/src/plugins/bi_import/Makefile.in	2011-05-13 14:53:35.507685313 -0400
@@ -459,6 +459,7 @@ AM_CFLAGS = \
   -I${top_srcdir}/lib/libc \
   ${GNOME_CFLAGS} \
   ${GLADE_CFLAGS} \
+  ${GUILE_CFLAGS} \
   ${QOF_CFLAGS} \
   ${GLIB_CFLAGS}
 
--- gnucash-2.4.12/src/import-export/aqbanking/Makefile.in.orig	2013-04-14 20:01:02.782526327 +0100
+++ gnucash-2.4.12/src/import-export/aqbanking/Makefile.in	2013-04-14 20:03:13.494518689 +0100
@@ -503,7 +503,7 @@
 	-I${top_srcdir}/src/gnome-utils -I${top_srcdir}/src/app-utils \
 	-I${top_srcdir}/src/engine -I${top_srcdir}/src/core-utils \
 	-I${top_srcdir}/src/gnc-module -I${top_srcdir}/src/libqof/qof \
-	${GNOME_CFLAGS} ${GLADE_CFLAGS} ${GLIB_CFLAGS} \
+	${GNOME_CFLAGS} ${GLADE_CFLAGS} ${GLIB_CFLAGS} ${GUILE_CFLAGS} \
 	${AQBANKING_CFLAGS} -DG_LOG_DOMAIN=\"gnc.import.aqbanking\"
 uidir = $(GNC_UI_DIR)
 ui_DATA = \
--- gnucash-2.4.12/src/plugins/bi_import/Makefile.in.orig	2013-04-14 20:01:02.905526319 +0100
+++ gnucash-2.4.12/src/plugins/bi_import/Makefile.in	2013-04-14 20:04:02.433515830 +0100
@@ -483,7 +483,7 @@
 	-I${top_srcdir}/src/gnome-utils -I${top_srcdir}/src/app-utils \
 	-I${top_srcdir}/src/engine -I${top_srcdir}/src/core-utils \
 	-I${top_srcdir}/src/gnc-module -I${top_srcdir}/lib/libc \
-	${GNOME_CFLAGS} ${GLADE_CFLAGS} ${QOF_CFLAGS} ${GLIB_CFLAGS} \
+	${GNOME_CFLAGS} ${GLADE_CFLAGS} ${GUILE_CFLAGS} ${QOF_CFLAGS} ${GLIB_CFLAGS} \
 	-DG_LOG_DOMAIN=\"gnc.plugin.bi-import\"
 all: all-recursive
 
