--- gnucash-2.4.12/gnucash-docs-2.2.0/Makefile.in.orig	2013-07-09 23:09:05.526166096 +0100
+++ gnucash-2.4.12/gnucash-docs-2.2.0/Makefile.in	2013-07-09 23:09:17.160165416 +0100
@@ -35,7 +35,7 @@
 	$(top_srcdir)/configure AUTHORS COPYING ChangeLog INSTALL NEWS \
 	config.guess config.sub install-sh ltmain.sh missing
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
-am__aclocal_m4_deps = $(top_srcdir)/configure.in
+am__aclocal_m4_deps = $(top_srcdir)/configure.ac
 am__configure_deps = $(am__aclocal_m4_deps) $(CONFIGURE_DEPENDENCIES) \
 	$(ACLOCAL_M4)
 am__CONFIG_DISTCLEAN_FILES = config.status config.cache config.log \
